cli_tools/gce_ovf_export/exporter/param_populator.go

```diff
@@ -20,13 +20,13 @@ import (
 	"strings"
 	"time"
 
-	"github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+	"github.com/GoogleCloudPlatform/compute-daisy/compute"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/domain"
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/param"
 	pathutils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/path"
 	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/storage"
-	ovfexportdomain "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_export/domain"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_ovf_export/domain"
```

Comment:
in other cases you newly added this import-as name, in this one you removed it. i'm curious but can't spot why?

Reply:
Pushed an update that uses `ovfexportdomain` globally.

Background: `cli_tools` follow a convention of grouping imports with:

```
stlib
\n
third-party
\n
same repo
```

Now that daisy is moving out of this repo, it's become a third-party dependency. I re-formatted the imports, and it looks like I goofed on some of the imports of the  `ovfexportdomain`  package.